tests/pytest_tests/unit_tests/test_cli.py

```diff
@@ -57,33 +57,6 @@ def hosts(self):
     monkeypatch.setattr(netrc, "netrc", lambda *args: FakeNet())
 
 
-@pytest.mark.skip(reason="Currently dont have on in cling")
-def test_enable_on(runner, git_repo):
-    with runner.isolated_filesystem():
-        with open("wandb/settings", "w") as f:
-            f.write("[default]\nproject=rad")
-        result = runner.invoke(cli.on)
-        print(result.output)
-        print(result.exception)
-        print(traceback.print_tb(result.exc_info[2]))
-        assert "W&B enabled" in str(result.output)
-        assert result.exit_code == 0
-
-
-@pytest.mark.skip(reason="Currently dont have off in cling")
-def test_enable_off(runner, git_repo):
-    with runner.isolated_filesystem():
-        with open("wandb/settings", "w") as f:
-            f.write("[default]\nproject=rad")
-        result = runner.invoke(cli.off)
-        print(result.output)
-        print(result.exception)
-        print(traceback.print_tb(result.exc_info[2]))
-        assert "W&B disabled" in str(result.output)
-        assert "disabled" in open("wandb/settings").read()
-        assert result.exit_code == 0
-
-
```

Comment:
we use offline/online instead in cli, these are hidden, so i'm removing tests are skipped anyways. ideally we would just remove the commands as well

Reply:
hmm... maybe we just enable it... ok... i'll undo this change